OSD.adoc

```diff
@@ -1,14 +1,16 @@
 ---
-layout: OSD-2022
-title: Open Source Day 2022
-permalink: /OSD-2022/
+layout: OSD
+title: Open Source Day 2023
+permalink: /OSD/
 image:
-  path: https://wildfly-security.github.io/wildfly-elytron/assets/images/posts/OpenSourceDayProject2022.png
+  path: https://prarthonapaul.github.io/wildfly-elytron/assets/images/posts/OSD2023Banner.png
```

Comment:
Once @Skyllarr has reviewed this, we can switch this back to wildfly-security.

Reply:
@fjuma @PrarthonaPaul  was just wondering why was the logo cut on the right side? there is a purple arrow on the right side in the original

Reply:
good catch, @PrarthonaPaul are you able to fix that?

Reply:
fixed! just waiting for the picture to update. You should see a black bar for now, but my last push removed it.